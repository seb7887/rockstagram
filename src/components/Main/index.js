import React from 'react';
import Header from '../Header';

const Main = ({ children }) => (
  <>
    <Header />
    {children}
  </>
);

export default Main;
