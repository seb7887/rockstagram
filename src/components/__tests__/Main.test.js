/* eslint-env jest */
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from 'react-testing-library';

import Main from '../Main';

const Children = () => <div>Children</div>;

describe('<Main/>', () => {
  it('renders and matches snapshot', () => {
    const { asFragment } = render(
      <Router>
        <Main>
          <Children />
        </Main>
      </Router>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
