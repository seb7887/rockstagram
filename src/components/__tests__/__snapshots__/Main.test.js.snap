// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> renders and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="HeaderStyles__Container-sc-18ybl58-0 brYbQH"
  >
    <h1>
      <a
        href="/"
      >
        Rockstagram
      </a>
    </h1>
  </div>
  <div>
    Children
  </div>
</DocumentFragment>
`;
